COMET/score_comet.py

```diff
@@ -7,42 +7,54 @@
 from comet import download_model, load_from_checkpoint
 
 
+pair2ref = {("wmt21.tedtalks", "en-de"): "refA", ("wmt21.tedtalks", "zh-en"): "refB",
+               ("wmt21.tedtalks", "en-ru"): "refA",
+               ("wmt21.news", "en-de"): "refC", ("wmt21.news", "zh-en"): "refB",
+               ("wmt21.news", "en-ru"): "refA"}
+
+# the code assumes the following directory structure:
```

Comment:
@GeorgeVern can we not keep the previous interface where the user specifies each file instead of a directory structure which is IMO hard to navigate?

Reply:
This is fixed now!